util/flag_parsers_test.go

```diff
@@ -0,0 +1,44 @@
+package util
+
+import (
+	"testing"
+)
+
+func TestMustParseIngestAfterFlag(t *testing.T) {
+	tests := []struct {
+		name              string
+		ingestAfterStr    string
+		expectedOrgID     uint32
+		expectedTimestamp int64
+		shouldPanic       bool
+	}{
+		{"empty", "", 0, 0, false},
+		{"only zeroes", "0:0", 0, 0, false},
+		{"typical", "10:4738913", 10, 4738913, false},
+		{"missing timestamp", "0", 0, 0, true},
+		{"text no number 1", "text", 0, 0, true},
+		{"text no number 2", "1:text", 0, 0, true},
+		{"orgid overflow", "10000000000:0", 0, 0, true},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			defer func() {
+				r := recover()
```

Comment:
have a look at : https://golang.org/src/regexp/regexp.go?s=9796:9832#L259
it seems a go best practice to have Must* style functions call regular functions that return an error.
that way in unit tests you can call the non-Must (non-panicing) version and just check for errors, rather than needing recover logic. this seems cleaner and simpler.

Reply:
> have a look at : https://golang.org/src/regexp/regexp.go?s=9796:9832#L259
> it seems a go best practice to have Must* style functions call regular functions that return an error.
> that way in unit tests you can call the non-Must (non-panicing) version and just check for errors, rather than needing recover logic. this seems cleaner and simpler.

Done in a1c8bf1